data/json/mutations.json

```diff
@@ -2503,7 +2502,7 @@
         "id" : "PRED3",
         "name" : "Predator",
         "points" : 3,
-        "description" : "Let's not overthink things.  You get hungry, you find prey, you kill it and you eat it.",
+        "description" : "You consider yourself something other than human and no longer empathise with them.  Combat skills are easy to learn and maintain, but your critical thinking, so characteristic to these creatures, suffers: -1 INT.",
```

Comment:
So long as the descriptive format adequately hints at what the trait actually does, which seems to be the objective of the PR.